cryptoki/src/object.rs

```diff
@@ -136,6 +136,8 @@ pub enum AttributeType {
     Wrap,
     /// Indicates that the key can only be wrapped with a wrapping key that has the Trusted attribute
     WrapWithTrusted,
+    /// Wraps undefined and custom attribute types.
+    Unhandled(CK_ATTRIBUTE_TYPE),
```

Comment:
Hey @ilkerBaltaci , thanks for your PR :pray:!

I understand the idea behind the `Unhandled` types, but I think we strive to only expose in the `cryptoki` crate idiomatic structures and functions which are safe to use and abstracted away from the raw PKCS11 types. That's summarized in our README:

> All the PKCS11 items might not be implemented but everything that is implemented is safe.

I will let other maintainers give their opinions but I think  that if you have new types/algos you want to add it's best to do so in the high-level "way" where you add corresponding `cryptoki` structures/functions for those new types